pypesto/visualize/optimizer_history.py

```diff
@@ -33,44 +45,44 @@ def optimizer_history(
 
     Parameters
     ----------
-    results: pypesto.Result or list
+    results:
         Optimization result obtained by 'optimize.py' or list of those
-    ax: matplotlib.Axes, optional
+    ax:
         Axes object to use.
     size: tuple, optional
         Figure size (width, height) in inches. Is only applied when no ax
         object is specified
-    trace_x: str, optional
+    trace_x:
         What should be plotted on the x-axis?
-        Possibilities: 'time', 'steps'
-        Default: 'steps'
-    trace_y: str, optional
+        Possibilities: TRACE_X
+        Default: TRACE_X_STEPS
+    trace_y:
         What should be plotted on the y-axis?
-        Possibilities: 'fval', 'gradnorm', 'stepsize'
-        Default: 'fval'
-    scale_y: str, optional
+        Possibilities: TRACE_Y
```

Comment:
not sure whether this will be displayed correctly as the tuple. If so everything fine, otherwise would change to
`TRACE_Y_FVAL, TRACE_Y_GRADNORM`